so-widgets-bundle.php

```diff
@@ -534,6 +534,32 @@ function get_widgets_list(){
 	}
 
 	/**
+	 * Get instances of all the widgets. Even ones that are not active.
+	 */
+	function get_widget_objects(){
```

Comment:
This could be quite a heavy call, couldn't it? Should the function perhaps be scoped private to try avoid abuse?